include/net/xia_ether.h

```diff
@@ -0,0 +1,36 @@
+#ifndef _NET_XIA_ETHER_H
+#define _NET_XIA_ETHER_H
```

Comment:
I think using a header file can help us keep the code organized, as one could review the main functions in ether.c and when looking for a deeper understanding of the code, they could also review the xia_ether.h file(especially for the one going to work on NWP part). If you still think that i should move everything except the Netlink stuff and XIDTYPE_ETHER to ether.c, please do comment.